processing/src/test/java/org/apache/druid/segment/join/JoinConditionAnalysisTest.java

```diff
@@ -220,6 +229,7 @@ public void test_forExpression_onlyRight()
         ImmutableList.of(),
         exprsToStrings(analysis.getNonEquiConditions())
     );
+    Assert.assertEquals(analysis.getRightEquiConditionKeys(), ImmutableSet.of("x"));
```

Comment:
These are backwards. (I think putting actual before expected is more natural, but JUnit disagrees and who am I to judge?)